### PR TITLE
REFACTOR: Import `visit` helper rather than using a global variable

### PR DESCRIPTION
**Pull request by eviltrout**

### CODE REVIEW

File: app/assets/javascripts/discourse-loader.js [hunk 1]
```diff
@@ -146,9 +146,6 @@ var define, requirejs;
       "@ember/object/internals": {
         guidFor: Ember.guidFor,
       },
-      "@ember/test-helpers": {
```
Comment (jjaffeux) [inline app/assets/javascripts/discourse-loader.js:149]: Don't think this change should be part of this PR

File: .eslintrc [hunk 2]
```diff
@@ -8,6 +8,7 @@
     "moduleForComponent": "off",
     "testStart": "off",
     "testDone": "off",
-    "sinon": "off"
+    "sinon": "off",
+    "visit": "off",
```
Comment (jjaffeux) [inline .eslintrc:12]: this should probably be in shared eslint config, but I will do it later as plugins will have to be updated too

Comment (eviltrout) [reply .eslintrc:12]: This was my mistake - I was used it to debug locally and find all occurrences. It causes test failures and I'm putting it back.

File: app/assets/javascripts/discourse-loader.js (hunk 1 above)
Comment (eviltrout) [reply app/assets/javascripts/discourse-loader.js:149]: It should be - it's defined now in `test-shims` including the visit helper. It doesn't need to be in the global loader anymore.

**Review by jjaffeux — APPROVED** [review]

**Pull request merged**